deployment/deployutils/deployutils.cpp

```diff
@@ -2004,6 +2004,21 @@ bool generateHardwareHeaders(const IPropertyTree* pEnv, StringBuffer& sbDefn, bo
       xpath.clear().append(XML_ATTR_TRACE);
       pNAS->addProp(xpath, sbdefaultValue.str());
     }
+
+    //<cost moneyLocale="en_US.UTF-8" perCpu="0.113" storageAtRest="0.0135" storageReads="0.0485" storageWrites="0.0038"/>
+
+    IPropertyTree* pCost = pCompTree->addPropTree("cost", createPTree());
+    xpath.clear().append("@memoryLocale");
```

Comment:
is this attribute supposed to be money locale rather than  memory locale?

Reply:
Shamser added it as a lower case. See https://github.com/hpcc-systems/HPCC-Platform/pull/15412/files.

I just duplicated it here.

Reply:
@kenrowland that seems unfortunate but not critical. At this point it's a matter of following the standards inherited in these files vs effort to change in the code. I'll leave it up to you and @shamser